lambda-audit-service/sample-app/function/lambda_audit_deployment_function.py

```diff
@@ -0,0 +1,106 @@
+import json
```

Comment:
Could you add more logging into this file? In addition, please document how can we find the relevant logs for this lambda function. We want to tell people how to debug when this is broken.